tests/cases/DI/OrmExtensionTest.php

```diff
@@ -51,13 +52,15 @@ public function testOwnEntityManager(): void
 			$compiler->addConfig([
 				'orm' => [
 					'entityManagerDecoratorClass' => DummyEntityManagerDecorator::class,
+					'configurationClass' => DummyConfiguration::class,
 				],
 			]);
 		}, self::class . __METHOD__);
 
 		/** @var Container $container */
 		$container = new $class();
 		self::assertInstanceOf(DummyEntityManagerDecorator::class, $container->getByType(DummyEntityManagerDecorator::class));
+		self::assertInstanceOf(DummyConfiguration::class, $container->getByType(DummyConfiguration::class));
```

Comment:
Please, add also test for the exception case. Thanks.

Reply:
Added.. can you help make test without warnings about deprecated anotations? I have no experience with PHPUnit.

Reply:
Replace annotation with method call.

https://github.com/nettrine/orm/commit/9d9413a97a9929bdc53f4592f98469962cfc354d#diff-9bd459dedc49d13726a368c5307d2fcaR41

Reply:
done :-)